testing/ase/test_mmio_c_ase.cpp

```diff
@@ -145,6 +156,8 @@ TEST(mmio_c_ase, offset_overflow) {
             FPGA_INVALID_PARAM);
```

Comment:
Not that this is related to your change but shouldn't this 64-bit test be passing in offsets that are 8 byte aligned (i.e. +8 instead of +4)?

Reply:
Some of the tests are to confirm that illegal address alignment triggers an error.